### ISSUE TITLE
The link in the About section of this repo is broken

### ISSUE THREAD

**Issue by jdaviescoates** [paviliondev/discourse-events#116]
This link

![image](https://user-images.githubusercontent.com/2113743/222100472-495d05a8-e088-48b7-8fe8-ea31e3ed4873.png)

Which is to https://plugins.discourse.pavilion.tech/c/discourse-events but get redirected to https://discourse.pluginmanager.org/c/discourse-events is broken.

![image](https://user-images.githubusercontent.com/2113743/222100396-fa6b61db-6554-4f95-870e-567bf604ef8a.png)

**Comment by nathan-nz** [comment]
Yup, there is a whole bunch of broken links at the mo due to moving the documentation over to coop.pavilion.tech. They should be fixed over the next week or so.
